packages/x-date-pickers-pro/src/internal/utils/valueManagers.ts

```diff
@@ -75,34 +76,43 @@ export const rangeFieldValueManager: FieldValueManager<
     const rawSectionsOfStartDate = getSections(start, prevDateRangeSections.startDate);
     const rawSectionsOfEndDate = getSections(end, prevDateRangeSections.endDate);
 
-    const sectionsOfStartDate = rawSectionsOfStartDate.map((section, sectionIndex) => {
-      if (sectionIndex === rawSectionsOfStartDate.length - 1) {
+    const sectionsOfStartDate: RangeFieldSectionWithoutPosition[] = rawSectionsOfStartDate.map(
+      (section, sectionIndex) => {
+        if (sectionIndex === rawSectionsOfStartDate.length - 1) {
+          return {
+            ...section,
+            dateName: 'start',
+            separator: `\u2069${' – '}\u2066`,
```

Comment:
Why do we need to use template string here ?
Is there a difference with this ?

```suggestion
            separator: '\u2069 – \u2066',
```

Reply:
Just the result of iterating on different solutions 🙈